recipes/loguru/all/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+cmake_minimum_required(VERSION 3.25)
```

Comment:
please, submit this cmake script to the upstream first.

Reply:
Upstream already has a PR related to adding full CMake build: https://github.com/emilk/loguru/pull/215, but it's not merged since April.

I can use the CMake script from this un-merged PR. But in the meantime I've decided to use a minimal own CMake script only for Conan package, and migrate to the "official" CMake build once upstream's PR-215 will be merged.